docs/tutorials/share-data-and-events-between-custom-functions-and-the-task-pane-tutorial.md

```diff
@@ -1,100 +1,132 @@
+You'll use the [Yeoman generator for Office Add-ins](https://github.com/OfficeDev/generator-office) to create the Excel add-in project.
 
-## Create the add-in project
+- To generate an Excel add-in with custom functions, run the command `yo office --projectType excel-functions --name 'Excel shared runtime add-in' --host excel --js true`.
 
-Use the Yeoman generator to create an Excel add-in project. Run the following command and then answer the prompts with the following answers.
+The generator will create the project and install supporting Node components.
 
-```command line
-yo office
-```
+## Configure the manifest
 
-- Choose a project type: **Excel Custom Functions Add-in project**
-- Choose a script type: **JavaScript**
-- What do you want to name your add-in? **My Office Add-in**
+Follow these steps to configure the add-in project to use a shared runtime.
 
-![Screenshot showing the prompts and answers for the Yeoman generator in a command line interface.](../images/yo-office-excel-project.png)
+1. Start Visual Studio Code and open the add-in project you generated.
+1. Open the **manifest.xml** file.
+1. Update the requirements section to use the [shared runtime](../reference/requirement-sets/shared-runtime-requirement-sets.md) instead of the custom function runtime. The XML should appear as follows.
 
-After you complete the wizard, the generator creates the project and installs supporting Node components.
+    ```xml
+    <Hosts>
+      <Host Name="Workbook"/>
+    </Hosts>
+    <Requirements>
+      <Sets DefaultMinVersion="1.1">
+        <Set Name="SharedRuntime" MinVersion="1.1"/>
+      </Sets>
+    </Requirements>
+    <DefaultSettings>
+    ```
 
-## Configure the manifest
+1. Find the `<VersionOverrides>` section and add the following `<Runtimes>` section. The lifetime needs to be **long** so that your add-in code can run even when the task pane is closed. The `resid` value is **Taskpane.Url**, which references the **taskpane.html** file location specified in the ` <bt:Urls>` section near the bottom of the **manifest.xml** file.
 
-1. Start Visual Studio Code and open the **My Office Add-in** project.
-2. Open the **manifest.xml** file.
-3. Find the `<VersionOverrides>` section, and add the following `<Runtimes>` section. The lifetime needs to be **long** so that the custom functions can still work even when the task pane is closed.
+    > [!IMPORTANT]
+    > The `<Runtimes>` section must be entered after the `<Host>` element in the exact order shown in the following XML.
 
    ```xml
-   <VersionOverrides xmlns="http://schemas.microsoft.com/office/taskpaneappversionoverrides" xsi:type="VersionOverridesV1_0">
+   <VersionOverrides ...>
      <Hosts>
-       <Host xsi:type="Workbook">
+       <Host ...>
          <Runtimes>
-           <Runtime resid="ContosoAddin.Url" lifetime="long" />
+           <Runtime resid="Taskpane.Url" lifetime="long" />
          </Runtimes>
-       <AllFormFactors>
+       ...
+       </Host>
```

Comment:
It's a little confusing that this `<Host>` element doesn't match the one in the previous step.

Reply:
Well, unfortunately, that is how the manifest is designed. This is actually a different section. Again why I try to include more details to help know where you are making updates.

Reply:
... I may have messed up this manifest, which would explain the error. Let me try again, but better!